packages/headless/src/app/commerce-engine/commerce-engine-configuration.ts

```diff
@@ -40,24 +41,49 @@ export const commerceEngineConfigurationSchema =
 
 export function getSampleCommerceEngineConfiguration(): CommerceEngineConfiguration {
```

Comment:
I copied this configuration over from atomic. This access token was already exposed in our documentation. https://github.com/coveo/ui-kit/blob/7aa6094d4bf63a81ae88c7327f84cc808f92f4b6/packages/atomic/storybookUtils/commerce-interface-wrapper.tsx#L10-L17

Reply:
I don't really fancy exposing a dev org....
At the very least, let's put a TODO with KIT-3244

Reply:
Would you rather we expose the prod barca org? It also has a public access token.

Nonetheless, I'll add a TODO with KIT-3244.

Reply:
We decided to keep the dev org for now to not potentially mess up barca prod analytics.